deploy/helm/templates/trivy-server.yaml

```diff
@@ -109,13 +109,10 @@ spec:
             - mountPath: /home/scanner/.cache
               name: data
               readOnly: false
+          {{- with .Values.trivy.resources }}
```

Comment:
maybe its better to have for it a separate value  `trivy.serverResources ` instead of `trivy.resources` as `server` resources need might be different from `scanJob` :
example :
```
serverResources:
  requests:
    cpu: 200m
    memory: 512Mi
  limits:
    cpu: 1
    memory: 1Gi
```
wdyt?

Reply:
Is it possible to use values. resources

Reply:
It will the best to create another value as suggested above

Reply:
Is trivy.server.resources more reasonable